tools/demo_server/config/metashell.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "name": "metashell",
+    "git": "https://github.com/sabel83/metashell.git",
+    "build": [
+      [
+        "/bin/bash",
+        "-c",
+        "git log -1 --pretty=\"%h %B\" | head -1 | sed 's/\\\\/\\\\\\\\/g' | sed 's/\"/\\\\\"/g' | sed 's/^/\"/' | sed 's/$/\\\\n This demo is supported by the Eotvos Lorand University, Faculty of Informatics\"/' > lib/core/version_desc.hpp"
```

Comment:
We could move the "supporter" text into a local file (outside of the git repository) and make the build pick it up from that.